Userland/killall.cpp

```diff
@@ -0,0 +1,95 @@
+#include <unistd.h>
+#include <stdio.h>
+#include <signal.h>
+#include <stdlib.h>
+#include <LibCore/CFile.h>
+#include <AK/AKString.h>
+
+static unsigned parse_uint(const String& str, bool& ok)
+{
+    if (str.is_empty()) {
+        ok = false;
+        return 0;
+    }
+    unsigned value = 0;
+    for (int i = 0; i < str.length(); ++i) {
+        if (str[i] < '0' || str[i] > '9') {
+            ok = false;
+            return 0;
+        }
+        value = value * 10;
+        value += (unsigned)(str[i] - '0');
+    }
+    ok = true;
+    return value;
+}
+
+static void print_usage_and_exit()
+{
+    printf("usage: killall [-signal] process_name\n");
+    exit(1);
+}
+
+static int kill_all(const String & process_name, const unsigned signum)
```

Comment:
Project coding style puts & together with the type like so:
const String& process_name